workflow/usage.rst

```diff
@@ -155,7 +155,16 @@ like this:
     The ``type`` (default value ``single_state``) and ``arguments`` (default value ``marking``)
     attributes of the ``marking_store`` option are optional. If omitted, their default values
     will be used.
+    
+.. note::
+
+    If a ``multiple_state`` is used, the persisted state type should be an ``array``, 
+    using Doctrine, you can use ``json`` or ``json_array``. 
+
+    If a ``single_state`` is used, the persisted state type could be a string
```

Comment:
Or it could be also an integer, but you need to implement a MarkingStoreInterface to convert from string to int the place

Reply:
Thanks for the feedback, I've added new lines about this case, can you review it and tell me if it's adapted? 🤔 

Thanks 😊